ipatests/test_integration/test_multidomain_ipa.py

```diff
@@ -0,0 +1,47 @@
+from ipatests.pytest_ipa.integration import tasks
+from ipatests.test_integration.base import MultiDomainIntegrationTest
+
+
+class TestMultidomain(MultiDomainIntegrationTest):
+    num_clients = 1
+    num_replicas = 1
+    num_trusted_clients = 1
+    num_trusted_replicas = 1
+    topology = 'line'
+
+    def test_multidomain_trust(self):
+        """
+        Test services on multidomain topology.
+        """
+
+        for host in (self.master, self.replicas[0],
+                     self.trusted_master, self.trusted_replicas[0]
+                     ):
+            tasks.start_ipa_server(host)
+
+        for host in (self.master, self.replicas[0],
+                     self.trusted_master, self.trusted_replicas[0],
+                     self.clients[0], self.trusted_clients[0]
+                     ):
+            tasks.kinit_admin(host)
+
+        # Add DNS forwarder to trusted domain on ipa domain
+        self.master.run_command([
+            "ipa", "dnsforwardzone-add", self.trusted_master.domain.name,
+            "--forwarder", self.trusted_master.ip,
+            "--forward-policy=only"
+        ])
+        self.trusted_master.run_command([
+            "ipa", "dnsforwardzone-add", self.master.domain.name,
+            "--forwarder", self.master.ip,
+            "--forward-policy=only"
+        ])
+
+        # Establish trust
+        self.master.run_command([
```

Comment:
You can comment out this command for now.

Reply:
Commenting out as this is tested in IDMCI
2024-10-24T14:50:29+0000 PASSED ipatests/test_integration/test_multidomain_ipa.py::TestMultidomain::test_multidomain_trust
2024-10-24T14:50:29+0000 ================== 1 passed, 1 warning in 2883.81s (0:48:03) ===================